project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.views_default.inc

```diff
@@ -0,0 +1,251 @@
+  $handler->display->display_options['fields']['c4m_media']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['c4m_media']['element_default_classes'] = FALSE;
+  $handler->display->display_options['fields']['c4m_media']['click_sort_column'] = 'fid';
+  $handler->display->display_options['fields']['c4m_media']['settings'] = array(
+    'image_style' => 'thumbnail',
+    'image_link' => '',
+  );
+  /* Field: Content: Status */
+  $handler->display->display_options['fields']['c4m_og_status']['id'] = 'c4m_og_status';
+  $handler->display->display_options['fields']['c4m_og_status']['table'] = 'field_data_c4m_og_status';
+  $handler->display->display_options['fields']['c4m_og_status']['field'] = 'c4m_og_status';
+  $handler->display->display_options['fields']['c4m_og_status']['label'] = '';
+  $handler->display->display_options['fields']['c4m_og_status']['exclude'] = TRUE;
+  $handler->display->display_options['fields']['c4m_og_status']['element_label_colon'] = FALSE;
+  /* Field: Global: Custom text */
+  $handler->display->display_options['fields']['nothing']['id'] = 'nothing';
+  $handler->display->display_options['fields']['nothing']['table'] = 'views';
+  $handler->display->display_options['fields']['nothing']['field'] = 'nothing';
+  $handler->display->display_options['fields']['nothing']['label'] = '';
+  $handler->display->display_options['fields']['nothing']['alter']['text'] = '<div class="related-block">
+[c4m_media]
+<div>
+<h3>[title]</h3>
+<div class="related-block-node-author">Owned by [c4m_first_name] [c4m_last_name] | [c4m_og_status]</div> 
+</div>
+</div>';
+  $handler->display->display_options['fields']['nothing']['element_label_colon'] = FALSE;
+  /* Sort criterion: Content: Post date */
+  $handler->display->display_options['sorts']['created']['id'] = 'created';
+  $handler->display->display_options['sorts']['created']['table'] = 'node';
+  $handler->display->display_options['sorts']['created']['field'] = 'created';
+  $handler->display->display_options['sorts']['created']['order'] = 'DESC';
+  /* Contextual filter: Content: Nid */
+  $handler->display->display_options['arguments']['nid']['id'] = 'nid';
+  $handler->display->display_options['arguments']['nid']['table'] = 'node';
+  $handler->display->display_options['arguments']['nid']['field'] = 'nid';
+  $handler->display->display_options['arguments']['nid']['relationship'] = 'reverse_c4m_related_projects_node';
+  $handler->display->display_options['arguments']['nid']['default_action'] = 'default';
+  $handler->display->display_options['arguments']['nid']['default_argument_type'] = 'og_context';
+  $handler->display->display_options['arguments']['nid']['summary']['number_of_records'] = '0';
+  $handler->display->display_options['arguments']['nid']['summary']['format'] = 'default_summary';
+  $handler->display->display_options['arguments']['nid']['summary_options']['items_per_page'] = '25';
```

Comment:
There's no limit defined. Since it's handpicked I would allow all to be displayed.